services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeService.kt

```diff
@@ -10,11 +10,7 @@ interface HankeService {
 
     fun loadHanke(hankeTunnus: String): Hanke?
 
-    fun findHankeOrThrow(hankeTunnus: String): Hanke
-
-    fun getHankeId(hankeTunnus: String): Int?
-
-    fun getHankeIdOrThrow(hankeTunnus: String): Int
+    fun findIds(hankeTunnus: String): HankeIds?
```

Comment:
Could the returned object be renamed to something like HankeIdentifier? This made me confused as I thought it returning some sort of list of id by hankeTunnus.

Reply:
Good point, there were already variables named `hankeIds` that were lists of ids of Hanke.